bldsys/cmake/sample_helper.cmake

```diff
@@ -184,6 +203,21 @@ function(add_sample_with_tags)
         list(APPEND SRC_FILES ${TARGET_FILES})
     endif()
 
+    # Gather GLSL shader files
+    set(SHADER_DIR_GLSL "${PROJECT_SOURCE_DIR}/shaders/${TARGET_ID}")
+    file(GLOB SHADERS_GLSL 
+        "${SHADER_DIR_GLSL}/*.vert"
+        "${SHADER_DIR_GLSL}/*.frag"
+        "${SHADER_DIR_GLSL}/*.comp"
+        "${SHADER_DIR_GLSL}/*.geom"
+        "${SHADER_DIR_GLSL}/*.tesc"
+        "${SHADER_DIR_GLSL}/*.tese"
+        "${SHADER_DIR_GLSL}/*.mesh"
+        "${SHADER_DIR_GLSL}/*.task"
+        "${SHADER_DIR_GLSL}/*.rgen"
+        "${SHADER_DIR_GLSL}/*.rchit"
+        "${SHADER_DIR_GLSL}/*.rmiss"
+    )
```

Comment:
Curious if this actually works as expected? I am not sure how this `${PROJECT_SOURCE_DIR}` works when used before the `add_project` call (this makes a `project` call later)?

Also nothing else seems to be dependent on these `add_sample*` functions. So perhaps a better option would be to move this outside to its own function? Will save on the repetition of code and offer more freedom if we want to make this logic more complex, looking for shaders in other places for example.